Contact/Contact Class and Smaller Related Data Classes/PhoneNumber.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Nate.ContactApp
+{
+    public class PhoneNumber
+    {
+        private string phoneNumber;
+        public string Number
+        {
+            get => phoneNumber;
+            set => phoneNumber = StripPhoneFormatting(value);
+        }
+
+        public PhoneNumber()
+        {
+            phoneNumber = "";
+        }
+
+        private string StripPhoneFormatting(string number)
```

Comment:
could use an extension method like we talked about but this is ok

Reply:
I chose to go this route so that any code related to handling phone numbers would be found in a single location, hopefully making things easier to track later on.